iq/iq.go

```diff
@@ -183,6 +184,33 @@ func validateRequiredOption(options Options, optionName string) (err error) {
 	return
 }
 
+// Audit accepts a slice of packageurl.PackageURL and will submit these to
+// Nexus IQ Server for audit, and return a struct of StatusURLResult
+func (i *Server) Audit(purls []packageurl.PackageURL) (StatusURLResult, error) {
+	i.logLady.WithFields(logrus.Fields{
+		"purls":          purls,
+		"application_id": i.Options.Application,
+	}).Info("Beginning audit with IQ using packageurl.PackageURL")
+
+	if i.Options.User == "admin" && i.Options.Token == "admin123" {
+		i.logLady.Info("Warning user of questionable life choices related to username and password")
+		warnUserOfBadLifeChoices()
+	}
+
+	internalID, err := i.getInternalApplicationID(i.Options.Application)
+	if internalID == "" && err != nil {
+		i.logLady.Error("Internal ID not obtained from Nexus IQ")
+		return statusURLResp, err
+	}
+
+	dx := cyclonedx.Default(i.logLady)
+
+	sbom := dx.FromPackageURLs(purls)
+	i.logLady.WithField("sbom", sbom).Debug("Obtained cyclonedx SBOM")
+
+	return i.getStatusURL(internalID, sbom)
```

Comment:
Probably only due to my lack of understanding of how this function will be used, but I wanted to ask: Does this function intentionally NOT call OSSI before IQ (the way the `AuditPackages()` function does? 

If it needed to call OSSI, then would probably need stuff like this too: https://github.com/sonatype-nexus-community/go-sona-types/pull/11/files#diff-4a23520ddfb008b69b217be292ad1880cdc5ce6b72a6ed4c69ebd7e13645fe76R242

Might be worth a short comment in the code about why/why not, and the use case, if only because there is a lot of similarity between these two functions.